### PR TITLE
update gallery images for optimization, commented out ad code that wa…

### PR DESCRIPTION
**Pull request by kimlarocca**
updated gallery images for optimization, commented out ad code that was causing the page to become unresponsive

https://jira.wnyc.org/browse/GOTH-187
https://jira.wnyc.org/browse/GOTH-199

Matt can you take a look and see what the ad issue might be? I commented out the code and that made the page work on local again.

### CODE REVIEW

**Comment by walsh9** [comment]
Took a quick look to see what was going on with the unresponsive gallery pages, but it's still happening for me on this branch.

Open home page, Clicked "Test Corona Preview", Opened gallery by clicking "View All 4", Images don't load and page is unresponsive...

**Comment by kimlarocca** [comment]
I did some research and found an issue with Sentry causing a recursion loop, I updated the sentry config and that seems to have fixed it... hopefully...

File: pages/_section/photos/_gallery.vue [hunk 1]
```diff
@@ -149,11 +151,11 @@ export default {
   },
   mounted () {
     // support deep linking
-    if (this.$route.query.image && this.$refs['image' + this.$route.query.image] !== undefined) {
-      const imageElement = this.$refs['image' + this.$route.query.image]
-      const top = imageElement[0].offsetTop
-      window.scrollTo(0, top + 72)
-    }
+    // if (this.$route.query.image && this.$refs['image' + this.$route.query.image] !== undefined) {
```
Comment (walsh9) [inline pages/_section/photos/_gallery.vue:154]: If the issue was the sentry thing does that mean we can uncomment this?

**Review by walsh9 — APPROVED** [review]

**Comment by kimlarocca** [comment]
You're right, forgot to un-comment that! It's still working fine, thanks for catching this :)

**Pull request merged**